### DISCUSSION TITLE
Integrative pseudo-time analyses

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#607] [General]
Is there a way to increase the number of features displayed in the integrated pseudotime heatmaps?

**Comment by rcorces** [comment]
I'm not sure if this is helpful but the parameters `varCutOff` and `maxFeatures` to the `plotTrajectoryHeatmap()` function might be relevant? Perhaps you can provide more information on what you are trying to accomplish.

**Comment by sylestiel** [comment]
Just to see an expanded repertoire.

**Reply by rcorces** [reply]
I believe you are looking for the parameters mentioned above. Specifically:
```
varCutOff | The "Variance Quantile Cutoff" to be used for identifying the top variable features across the given trajectory. Only features with a variance above the provided quantile will be retained.
```
